<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumberWindow</class>
 <widget class="QDialog" name="NumberWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>MV Boli</family>
      </font>
     </property>
     <property name="text">
      <string>Number of players</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="ok_button">
     <property name="font">
      <font>
       <family>MV Boli</family>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="back_button">
     <property name="font">
      <font>
       <family>MV Boli</family>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="numberSpinBox">
     <property name="font">
      <font>
       <family>MV Boli</family>
      </font>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>7</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
